website/docs/r/is_virtual_endpoint_gateway.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: "ibm"
+page_title: "IBM : docs-ibm-resource-is-virtual-endpoint-gateway"
+sidebar_current: "ibm_is_virtual_endpoint_gateway"
+description: |-
+  Manages IBM Virtual endpoint gateway
+---
+
+# ibm\_is_virtual_endpoint_gateway
+
+Provides a Virtual endpoint gateway  resource. This allows Virtual endpoint gateway  to be created, updated, and cancelled.
+
+
+## Example Usage
+
+In the following example, you can create a VPN gateway:
+
+```hcl
+resource "ibm_is_virtual_endpoint_gateway" "endpoint_gateway1" {
+		
+  name = "my-endpoint-gateway-1"
+  target {
+	name          = "ibm-dns-server2"
+    resource_type = "provider_infrastructure_service"
+  }
+  vpc = ibm_is_vpc.testacc_vpc.id
+  resource_group = data.ibm_resource_group.test_acc.id    
+}
+
+resource "ibm_is_virtual_endpoint_gateway" "endpoint_gateway2" {
+	name = "my-endpoint-gateway-1"
+	target {
+	  name          = "ibm-dns-server2"
+	  resource_type = "provider_infrastructure_service"
+	}
+	vpc = ibm_is_vpc.testacc_vpc.id
+	ips {
+		subnet_id   = ibm_is_subnet.testacc_subnet.id
+		name        = "test-reserved-ip1"
+	}
+	resource_group = data.ibm_resource_group.test_acc.id
+}
+
+resource "ibm_is_virtual_endpoint_gateway" "endpoint_gateway3" {
+	name = "my-endpoint-gateway-1"
+	target {
+	  name          = "ibm-dns-server2"
+	  resource_type = "provider_infrastructure_service"
+	}
+	vpc = ibm_is_vpc.testacc_vpc.id
+	ips {
+		id   = "0737-5ab3c18e-6f6c-4a69-8f48-20e3456647b5"
+	}
+	resource_group = data.ibm_resource_group.test_acc.id
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, string,ForceNew) Endpoint gateway name
+* `target` - (Required, stringList) The unique identifier of VPN gateway(ID).
```

Comment:
The unique identifier of VPN gateway(ID). this defination is wrong

Reply:
handled